l10n_it_delivery_note/report/report_delivery_note.xml

```diff
@@ -0,0 +1,334 @@
+                                    </td>
+                                </t>
+                            </tr>
+                            <tr t-if="line.product_id.tracking != 'none'">
+                                <td colspan="99" style="font-size:10px;">
+                                    <t t-foreach="line.move_id.move_line_ids" t-as="lot_line">
+                                        lot / sn :
+                                        <strong>
+                                            <span t-field="lot_line.lot_id" />
+                                        </strong>
+                                        <t t-if="'use_date' in lot_line.lot_id.fields_get()">
+                                            <t t-if="lot_line.lot_id.use_date">
+                                                best before
+                                                <strong>
+                                                    <span t-field="lot_line.lot_id.use_date" t-options='{"format": "dd/MM/yyyy"}'/>
+                                                </strong>
+                                            </t>
+                                        </t>
+                                        <t t-if="line.product_qty != lot_line.qty_done">
+                                            qty
+                                            <span t-field="lot_line.qty_done"/>
+                                        </t>
+                                        ;
+                                    </t>
+                                </td>
+                            </tr>
+                        </t>
+                    </tbody>
+                </table>
+
+                <table class="table table-condensed table-bordered delivery_data">
+                    <tr>
+                        <td>
+                            <div name="good_description" style="font-size:10px;">Goods Description</div>
+                            <div style="font-size:12px;" t-field="doc.goods_appearance_id"/>
+                        </td>
+                        <td>
+                            <div name="gross_weight" style="font-size:10px;">Gross Weight</div>
+                            <div class="text-right" style="font-size:12px;" t-field="doc.gross_weight"/>
+                        </td>
+                        <td>
+                            <div name="net_weight" style="font-size:10px;">Net Weight</div>
+                            <div class="text-right" style="font-size:12px;" t-field="doc.net_weight"/>
+                        </td>
+                        <td>
+                            <div name="transport_date" style="font-size:10px;">DN date</div>
+                            <div class="m-0" style="font-size:12px;" t-field="doc.transport_datetime"/>
+                        </td>
+                        <td>
+                            <div name="parcel" style="font-size:10px;">Packages</div>
```

Comment:
Penso sia da modificare anche questa:

```suggestion
                            <div name="packages" style="font-size:10px;">Packages</div>
```

Reply:
Ciao @primes2h , 

questo merge sta diventando un incubo, fai mergiare il modulo poi c'è tempo per mettere a posto queste sfumature. 
fin quando apriremo request per virgole punti e non andremo mai avanti. 
Una volta mergiato si potranno fare pr a volontà. 

comunque Giuseppe lo sta sistemando. 

tieni presente che per fare questa modifica ci vuole circa 30 minuti perchè ci deve essere una PR con 4 commit e se la modifica richiesta è sul primo commit bisogna cancellare tutto e iniziare da zero , primo commit con la modifica, secondo commit con nulla cambiato ecc e poi push . Con il modulo mergiato invece è roba di pochi secondi che può fare chiunque.

Reply:
> tieni presente che per fare questa modifica ci vuole circa 30 minuti perchè ci deve essere una PR con 4 commit e se la modifica richiesta è sul primo commit bisogna cancellare tutto e iniziare da zero , primo commit con la modifica, secondo commit con nulla cambiato ecc e poi push . Con il modulo mergiato invece è roba di pochi secondi che può fare chiunque.

Se sono modifiche per correzioni di questo tipo potrebbe essere sufficiente anche un commit separato, che è più semplice da gestire.
Il fatto poi che la cosa si sia protratta a lungo è (in parte) dovuto anche al fatto che alcune revisioni iniziali erano rimaste aperte, e ho dovuto sollecitare più volte (anche le richieste di chiarimento). Il tempo è prezioso per tutti :wink: 
Comunque adesso è ok, grazie.